Darwin/Examples/StressTestApp/StressTestApp/ViewController.swift

```diff
@@ -0,0 +1,261 @@
+    private var syncCollection: Sync<Document>?
+    
+    // UI Elements
+    @IBOutlet weak var numSyncedDocsLabel: UILabel!
+    
+    @IBOutlet weak var numDocsToInsertInput: UITextField!
+    @IBOutlet weak var docSizeInput: UITextField!
+    
+    @IBOutlet weak var insertManyButton: UIButton!
+    @IBOutlet weak var clearAllDocsButton: UIButton!
+    @IBOutlet weak var syncPassButton: UIButton!
+    
+    @IBOutlet weak var timeLabel: UILabel!
+    @IBOutlet weak var networkSentLabel: UILabel!
+    @IBOutlet weak var networkReceivedLabel: UILabel!
+    
+    let appDelegate = UIApplication.shared.delegate as! AppDelegate
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+        let cornerRadius: CGFloat = 20;
+        insertManyButton.layer.cornerRadius = cornerRadius;
+        clearAllDocsButton.layer.cornerRadius = cornerRadius;
+        syncPassButton.layer.cornerRadius = cornerRadius;
+        
+        // Keyboard dismissal
+        let tap = UITapGestureRecognizer(target: self.view, action: #selector(UIView.endEditing(_:)))
+        tap.cancelsTouchesInView = false
+        self.view.addGestureRecognizer(tap)
+        
+        // Initialize Stitch RemoteMongoDBService and RemoteMongoCollection
+        do {
+            mongoClient = try stitchClient.serviceClient(fromFactory: remoteMongoClientFactory, withName: "mongodb-atlas")
+            mongoCollection = mongoClient?.db("stress").collection("tests");
+            syncCollection = mongoCollection?.sync
+            
+            syncCollection?.configure(
+                conflictHandler: DefaultConflictHandler<Document>.remoteWins(),
+                changeEventDelegate:  { documentId, event in
+                    self.log("Sync received changeEvent of type: \(event.operationType) and body: \(String(describing: event.fullDocument))")
+                    if (event.hasUncommittedWrites) {
+                        self.log("Sync changeEvent has uncommitted writes")
+                    }
+                }, errorListener: self.on(error:forDocumentId:)) { result in
+                    switch result {
+                    case .success:
+                        self.log("Successfully Configured sync");
+                        self.syncCollection?.proxy.dataSynchronizer.isSyncThreadEnabled = false
+                        self.syncCollection?.proxy.dataSynchronizer.stop()
+                    case .failure(let error):
+                        self.log("Failed to auth with error: \(error)");
+                    }
+                }
+            
+            stitchClient.auth.login(withCredential: AnonymousCredential()) { result in
+                switch result {
+                case .success(let user):
+                    self.log("Successfully authenticated with user: \(user.id)");
+                case .failure(let error):
```

Comment:
[nit] we may want to consider fatalErroring here. This is a sample app and this would be a rough state to be in.